packages/fxa-payments-server/src/routes/Subscriptions/SubscriptionItem.tsx

```diff
@@ -72,16 +88,31 @@ export const SubscriptionItem = ({
 
         {!customerSubscription.cancel_at_period_end ? (
           <>
-            <PaymentUpdateForm
-              {...{
-                plan,
-                customerSubscription,
-                customer,
-                updatePayment,
-                resetUpdatePayment,
-                updatePaymentStatus,
-              }}
-            />
+            {useSCAPaymentFlow ? (
```

Comment:
The changes here in SubscriptionItem are plumbing and a component switch on `useSCAPaymentFlow`